index.bs

```diff
@@ -3939,12 +3942,15 @@ JavaScript APIs.
 
 ## Simple Transaction Authorization Extension (txAuthSimple) ## {#sctn-simple-txauth-extension}
 
-This [=registration extension=] and [=authentication extension=] allows for a simple form of transaction authorization. A
+This extension allows for a simple form of transaction authorization. A
 [=[RP]=] can specify a prompt string, intended for display on a trusted device on the authenticator.
 
 : Extension identifier
 :: `txAuthSimple`
 
+: Operation applicability
+:: [=registration extension|Registration=] and [=authentication extension|authentication=]
```

Comment:
I would always capitalize both Registration and Authentication when used in this context.  So please change [=authentication extension|authentication=] to [=authentication extension|Authentication=] - both here and in all other occurrences.

Reply:
Done, thanks!